addons/io/org.openhab.io.neeo/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,61 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Neeo Transport
```

Comment:
This is actually not a "transport" bundle, but an I/O add-on. We don't have a fixed naming scheme for those, but e.g. Homekit and ImperiHome call them "Integration", so I'd suggest to go for the same: "NEEO Integration"